jwql/tests/test_preview_image.py

```diff
@@ -47,15 +52,44 @@ def test_directory(test_dir=TEST_DIRECTORY):
         Path to directory used for testing
 
     """
-    os.mkdir(test_dir)  # creates directory
+    # Set up local test directory
+    if not os.path.isdir(test_dir):
```

Comment:
There is an `ensure_dir_exists` function in utils.py that does this. It also sets the permissions correctly on the directory if the directory needs to be created.

Reply:
Ah, I completely forgot about that util. Thank you!